nilearn/datasets/func.py

```diff
@@ -73,9 +73,9 @@ def fetch_haxby_simple(data_dir=None, url=None, resume=True, verbose=1):
     # There is a common file for the two versions of Haxby
     fdescr = _get_dataset_descr('haxby2001')
 
-    # return the data
-    return Bunch(func=files[1], session_target=files[0], mask=files[2],
-                 conditions_target=files[3], description=fdescr)
+    # return the example data in a list
```

Comment:
Slightly inaccurate since you return a Bunch object and not a list. Say something about list being used because the dataset is single-subject.